src/probnum/diffeq/odefiltsmooth/ivpfiltsmooth.py

```diff
@@ -56,45 +54,57 @@ def solve(self, firststep, nsteps=1, **kwargs):
 
         ####### This function surely can use some code cleanup. #######
 
-        current = self.gfilt.initialdistribution
+        current_rv = self.gfilt.initialrandomvariable
         step = firststep
-        ssqest, ct = 0.0, 0
-        times, means, covars = [self.ivp.t0], [current.mean()], [current.cov()]
+        ssqest, n_steps = 0.0, 0
```

Comment:
How strong is your opinion on 'n_steps'? ;) I agree that it is better than `ct` (count), maybe something even better would be `num_steps`? Not a big issue though :)

Reply:
Not strong! I can definitely change that to `num_steps`. I'd also be fine with `count`, `n`, `n_iter`, or some other proposals.